mobile/src/bundles/common/common-api.ts

```diff
@@ -52,7 +52,7 @@ class TalentApi extends HttpApiBase {
 
     public async getUserDetailsByUserId(
         payload: Partial<UserDetailsGeneralRequestDto>,
-    ): Promise<UserDetailsGeneralRequestDto | null> {
+    ): Promise<UserDetailsResponseDto | null> {
         const { userId = '' } = payload;
         const response = await this.load(
             this.getFullEndpoint('/', userId, {}),
```

Comment:
Are you sure about this slash? We have removed all shashes at the end of endpoints

Reply:
This is a '/' before userId